src/data/links.yaml

```diff
@@ -1651,10 +1697,35 @@ links:
 
   - title: Climate Action Tracker
     url: https://climateactiontracker.org/countries/
-    tags: ['local-authorities']
+    tags: ["local-authorities"]
     categories:
       - carbon-reduction
     description: >
       See which countries are on track for the Paris Agreement or similar, and
       which countries are failing horribly. Hint: it's everyone but Morocco and
       The Gambia.
+
+  - title: Stay Grounded
+    url: https://stay-grounded.org/
+    categories: [travel]
+    tags: [flightfree]
+    description: >-
+      A global network of more than 150 member organisations, among them local
+      airport opposition and climate justice groups, NGOs, trade unions, initiatives
+      fostering alternatives to aviation like night trains, and organisations
+      supporting communities which struggle against offset projects or biofuel plantations
+
+  - title: Vojo
```

Comment:
They're talking about budgets rather than footprints I think. 

Budget is how much carbon we're _supposed_ to generate individually each year if we want to stay below 1.5c.

Agree with what you say about being pre-launch though.

Reply:
OH sorry budget, yeah, that didnt click with me and I need to introduce that term on the site. 

I got chatting to the vojo chap and they're delayed a bit in launch, but I'll absolutely add this one when they're live.